### PR TITLE
Update CHANGES to reflect already-merged PRs

### PR DESCRIPTION
**Pull request by ciaranmcnulty**
Because @stof said he may do a  release soon

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+  * Add ability to specify execution order of suite (thanks @ciaranmcnulty)
+  * Add translated keywords in definion printre (thanks @WouterJ)
+  * Add 'rowtable' transformations (thanks @PurpleBooth)
+  * Handle case when nonexistant config file is used (thanks @watermanio)
+  * Added Japanese translation (thanks @SNakano)
+  * Fix output buffering errors (thanks @tscheepers)
+  * Allow --verbose flag at CLI (thanks @pfrenssen)
+  * Allow hyphens in suite names (thanks @WouterJ)
+  * Improve performance of Turnip parsin (thanks @Sam-Burns)
```
Comment (stof) [inline CHANGES.md]: typo here. Missing ``g``

**Comment by Taluu** [comment]
I think you missed symfony's 3 compatiility (via #801)

**Comment by ciaranmcnulty** [comment]
Thanks @Taluu and @stof, updated

**Comment by everzet** [comment]
Thanks guys!

**Pull request merged**